.github/workflows/ci.yaml

```diff
@@ -0,0 +1,107 @@
+name: CI Tasks for Ory CLI
+on:
+  push:
+    branches:
+      - main
+      - master
+    tags:
+      - '*'
+  pull_request:
+
+jobs:
+  test:
+    name: Run tests
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request'
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '^1.17'
+      - run: make test
+
+  docs:
+    name: Generate docs
+    runs-on: ubuntu-latest
+    if: ${{ github.event.push.ref == 'refs/heads/master' }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '^1.17'
+      - run: .github/docs.sh
+
+  release:
+    name: Generate release
+    runs-on: ubuntu-latest
+    if: ${{ github.ref_type == 'tag' }}
+    needs:
+      - test
+    steps:
+      - uses: ory/ci/releaser@v1
+        with:
+          token: ${{ secrets.PAT }}
+          goreleaser_key: ${{ secrets.GORELEASER_KEY }}
+
+  npm-publish:
+    name: Publish to npm
+    runs-on: ubuntu-latest
+    if: ${{ github.ref_type == 'tag' }}
+    needs:
+      - release
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+      - run: |
+          npm install
+          npm run test:binwrap
+          npm version --no-git-tag-version ${{ github.ref_name }}
+          echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > ~/.npmrc
+          npm publish --access public
+
+  newsletter-draft:
+    name: Draft newsletter
+    runs-on: ubuntu-latest
+    if: ${{ github.ref_type == 'tag' }}
+    needs:
+      - release
+    steps:
+      - uses: ory/ci/newsletter@v1
+        with:
+          mailchimp_list_id: 058a056618
+          mailchmip_segment_id: 11398953
```

Comment:
I confirmed that these values are indeed the a test audience - good to go @icyphox